Sources/SwiftProtobuf/GetPathDecoder.swift

```diff
@@ -44,211 +46,214 @@ struct GetPathDecoder<T: Message>: Decoder {
     return number
   }
 
-  private mutating func captureValue(_ value: Any?) {
+  private mutating func captureValue(_ value: Any?) throws {
+    if !nextPath.isEmpty {
```

Comment:
This might read a little better as a `guard`.

```swift
guard nextPath.isEmpty else {
  throw PathDecodingError.pathNotFound
}
```

Reply:
Done, and also previous comments were fixed. I appreciate if you review them and mark them as resolved if they are fixed.